2022-05-12 22:25:36,359 [cuckoo.core.scheduler] INFO: Task #18: acquired machine 192.168.56.102 (label=192.168.56.102)
2022-05-12 22:25:36,360 [cuckoo.core.resultserver] DEBUG: Now tracking machine 192.168.56.101 for task #18
2022-05-12 22:25:36,360 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Replay
2022-05-12 22:25:36,376 [cuckoo.auxiliary.sniffer] INFO: Started sniffer with PID 4690 (interface=vboxnet0, host=192.168.56.101)
2022-05-12 22:25:36,376 [cuckoo.core.plugins] DEBUG: Started auxiliary module: Sniffer
2022-05-12 22:25:36,416 [cuckoo.machinery.virtualbox] DEBUG: Starting vm 192.168.56.102
2022-05-12 22:25:36,732 [cuckoo.machinery.virtualbox] DEBUG: Restoring virtual machine 192.168.56.102 to its current snapshot
2022-05-12 22:25:41,499 [cuckoo.core.guest] INFO: Starting analysis #18 on guest (id=192.168.56.102, ip=192.168.56.101)
2022-05-12 22:25:42,512 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 22:25:43,532 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 22:25:44,538 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 22:25:44,579 [cuckoo.core.guest] DEBUG: 192.168.56.102: not ready yet
2022-05-12 22:25:45,618 [cuckoo.core.guest] INFO: Guest is running Cuckoo Agent 0.10 (id=192.168.56.102, ip=192.168.56.101)
2022-05-12 22:25:45,741 [cuckoo.core.guest] DEBUG: Uploading analyzer to guest (id=192.168.56.102, ip=192.168.56.101, monitor=latest, size=3884763)
2022-05-12 22:25:50,132 [cuckoo.core.resultserver] DEBUG: Task #18: live log analysis.log initialized.
2022-05-12 22:25:53,445 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:25:58,508 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:03,575 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:08,606 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:11,619 [cuckoo.core.resultserver] DEBUG: Task #18 is sending a BSON stream
2022-05-12 22:26:13,277 [cuckoo.core.resultserver] DEBUG: Task #18 is sending a BSON stream
2022-05-12 22:26:13,696 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:14,224 [cuckoo.core.resultserver] DEBUG: Task #18: File upload for 'shots/0001.jpg'
2022-05-12 22:26:14,268 [cuckoo.core.resultserver] DEBUG: Task #18 uploaded file length: 47851
2022-05-12 22:26:18,793 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:23,886 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:28,938 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:33,981 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:39,100 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:44,176 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:47,615 [cuckoo.core.resultserver] DEBUG: Task #18: File upload for 'memory/2704-1.dmp'
2022-05-12 22:26:49,215 [cuckoo.core.guest] DEBUG: 192.168.56.102: analysis #18 still processing
2022-05-12 22:26:51,988 [cuckoo.core.resultserver] DEBUG: Task #18 uploaded file length: 69498432
2022-05-12 22:26:53,472 [cuckoo.core.resultserver] DEBUG: Task #18: File upload for 'shots/0002.jpg'
2022-05-12 22:26:53,639 [cuckoo.core.resultserver] DEBUG: Task #18 uploaded file length: 42889
2022-05-12 22:26:54,270 [cuckoo.core.guest] INFO: 192.168.56.102: analysis completed successfully
2022-05-12 22:26:54,286 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Replay
2022-05-12 22:26:54,288 [cuckoo.core.plugins] DEBUG: Stopped auxiliary module: Sniffer
2022-05-12 22:26:54,288 [cuckoo.machinery.virtualbox] DEBUG: Stopping vm 192.168.56.102
2022-05-12 22:26:54,563 [cuckoo.core.resultserver] DEBUG: Task #18 had connection reset for <Context for LOG>
2022-05-12 22:26:55,655 [cuckoo.core.resultserver] DEBUG: Stopped tracking machine 192.168.56.101 for task #18
2022-05-12 22:26:55,662 [cuckoo.core.scheduler] DEBUG: Released database task #18
2022-05-12 22:26:55,703 [cuckoo.core.plugins] DEBUG: Executed processing module "AnalysisInfo" for task #18
2022-05-12 22:26:55,708 [cuckoo.core.plugins] DEBUG: Executed processing module "BehaviorAnalysis" for task #18
2022-05-12 22:26:55,709 [cuckoo.core.plugins] DEBUG: Executed processing module "Dropped" for task #18
2022-05-12 22:26:55,709 [cuckoo.core.plugins] DEBUG: Executed processing module "DroppedBuffer" for task #18
2022-05-12 22:26:56,527 [cuckoo.core.plugins] DEBUG: Executed processing module "MetaInfo" for task #18
2022-05-12 22:26:58,509 [cuckoo.core.plugins] DEBUG: Executed processing module "ProcessMemory" for task #18
2022-05-12 22:26:58,510 [cuckoo.core.plugins] DEBUG: Executed processing module "Procmon" for task #18
2022-05-12 22:26:58,532 [cuckoo.core.plugins] DEBUG: Executed processing module "Screenshots" for task #18
2022-05-12 22:26:58,533 [cuckoo.core.plugins] DEBUG: Executed processing module "Static" for task #18
2022-05-12 22:26:58,538 [cuckoo.core.plugins] DEBUG: Executed processing module "Strings" for task #18
2022-05-12 22:26:58,540 [cuckoo.core.plugins] DEBUG: Executed processing module "TargetInfo" for task #18
2022-05-12 22:26:58,858 [cuckoo.core.plugins] DEBUG: Executed processing module "NetworkAnalysis" for task #18
2022-05-12 22:26:58,858 [cuckoo.core.plugins] DEBUG: Executed processing module "Extracted" for task #18
2022-05-12 22:26:58,859 [cuckoo.core.plugins] DEBUG: Executed processing module "TLSMasterSecrets" for task #18
2022-05-12 22:26:58,862 [cuckoo.core.plugins] DEBUG: Executed processing module "Debug" for task #18
2022-05-12 22:26:58,863 [cuckoo.core.scheduler] ERROR: Failure in AnalysisManager.run
Traceback (most recent call last):
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/scheduler.py", line 766, in run
    self.process_results()
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/scheduler.py", line 707, in process_results
    RunSignatures(results=results).run()
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/cuckoo/core/plugins.py", line 342, in __init__
    csv_data = pd.read_csv(csv_file, low_memory = False)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/cuckoo/cuckoo/local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
IOError: [Errno 2] File ioc_ip.csv does not exist: 'ioc_ip.csv'
